DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/bridging/test_bridging_unregistered_traffic_ipv6.py

```diff
@@ -0,0 +1,179 @@
+import pytest
+import asyncio
+
+from dent_os_testbed.lib.bridge.bridge_link import BridgeLink
+from dent_os_testbed.lib.ip.ip_link import IpLink
+
+from dent_os_testbed.utils.test_utils.tgen_utils import (
+    tgen_utils_get_dent_devices_with_tgen,
+    tgen_utils_traffic_generator_connect,
+    tgen_utils_dev_groups_from_config,
+    tgen_utils_get_traffic_stats,
+    tgen_utils_setup_streams,
+    tgen_utils_start_traffic,
+    tgen_utils_stop_traffic,
+    tgen_utils_get_loss,
+)
+
+pytestmark = [
+    pytest.mark.suite_functional_bridging,
+    pytest.mark.asyncio,
+    pytest.mark.usefixtures("cleanup_bridges", "cleanup_tgen")
+]
+
+async def test_bridging_unregistered_traffic_ipv6(testbed):
+    """
+    Test Name: test_bridging_unregistered_traffic_ipv6
+    Test Suite: suite_functional_bridging
+    Test Overview: Verify bridge flooding behaviour of unregistered IPv6 packets.
+    Test Author: Kostiantyn Stavruk
+    Test Procedure:
+    1.  Init bridge entity br0.
+    2.  Set ports swp1, swp2, swp3, swp4 master br0.
+    3.  Set entities swp1, swp2, swp3, swp4 UP state.
+    4.  Set bridge br0 admin state UP.
+    5.  Send increment traffic from TG.
+    6.  Verify that traffic flooded to all ports that are members in br0.
+    7.  Disable multicast flooding on the ports.
+    8.  Send increment traffic from TG.
+    9.  Verify that traffic was not flooded/forwarded to any of the disabled ports.
+    10. Enable back multicast flooding on the ports.
+    11. Send increment traffic from TG.
+    12. Verify that traffic flooded to all ports that are members in br0.
+    """
+
+    bridge = "br0"
+    tgen_dev, dent_devices = await tgen_utils_get_dent_devices_with_tgen(testbed, [], 4)
+    if not tgen_dev or not dent_devices:
+        pytest.skip("The testbed does not have enough dent with tgen connections")
+    dent_dev = dent_devices[0]
+    device_host_name = dent_dev.host_name
```

Comment:
dent_dev is not used later, change it to `device_host_name = dent_devices[0].host_name`